### DISCUSSION TITLE
Support for SSR in AstroWind Template

### DISCUSSION THREAD

**Discussion by divya-binil** [arthelokyo/astrowind#647] [Ideas]
While working on a project that requires token-based viewing of protected pages, I encountered a limitation with the AstroWind template when attempting to enable SSR (output: 'server' in astro.config.mjs). As confirmed in [Issue #194](https://github.com/onwidget/astrowind/issues/194), SSR is currently unsupported, and enabling it causes the project to fail.

Additionally, when changing the configuration to set prerender: false in combination with SSR, I encountered a UI error. I’ve attached a screenshot of the error below for reference.

![image](https://github.com/user-attachments/assets/2fba3551-ec5e-4fb4-aefe-295613903f74)

**Use Case**
Our project involves displaying sensitive, unpublished content that needs to be securely accessed using server-side rendering (SSR). This ensures that tokens and sensitive data are not exposed in the client-side code, which is a critical requirement for our application.

**Request**
It would be highly beneficial if AstroWind could support SSR, even with minimal configuration changes. Alternatively, detailed guidance or workarounds to enable SSR in the current version would be immensely helpful.

**Comment by alfadormx** [comment]
Hello, did you manage to solve this ?
